pkg/kubelet/cm/container_manager_windows.go

```diff
@@ -93,12 +95,21 @@ func (cm *containerManagerImpl) Start(ctx context.Context, node *v1.Node,
 
 	containerMap, containerRunningSet := buildContainerMapAndRunningSetFromRuntime(ctx, runtimeService)
 
-	// Initialize CPU manager
 	if utilfeature.DefaultFeatureGate.Enabled(kubefeatures.WindowsCPUAndMemoryAffinity) {
+		// Initialize CPU manager
 		err := cm.cpuManager.Start(cpumanager.ActivePodsFunc(activePods), sourcesReady, podStatusProvider, runtimeService, containerMap)
 		if err != nil {
 			return fmt.Errorf("start cpu manager error: %v", err)
 		}
+
+		// Initialize memory manager
+		if utilfeature.DefaultFeatureGate.Enabled(kubefeatures.MemoryManager) {
+			containerMap, _ := buildContainerMapAndRunningSetFromRuntime(ctx, runtimeService)
```

Comment:
you can now use the value returned on line 96 above, like the linux container manager does.